### ISSUE TITLE
example 8.2: calculating fraction lost vs fractionLost stat

### ISSUE THREAD

**Issue by fippo** [w3c/webrtc-stats#190]
the example in 8.2 essentially attempts to show a warning to the user if the packet loss is > 30%.
It calculates the fraction lost as
```
var fractionLost = (packetsSent - packetsReceived) / packetsSent;
```

Does that make sense given that we have a fraction loss
https://w3c.github.io/webrtc-stats/#dom-rtcinboundrtpstreamstats-fractionlost

Note that the fraction lost in the example is the fraction of packets lost since the last time getStats was called whereas the fractionlost on the inboundrtpstreamstats is probably the instantaneous value from the last receiver report.

**Comment by vr000m** [comment]
should probably call the variable intervalFractionLost. This will indeed be different from the factionLost reported by the getStats, which is the fraction of packets lost since sending the last receiver report.

**Comment by alvestrand** [comment]
We can fix this in #192 which is touching that code anyway.
